pytket/binders/passes.cpp

```diff
@@ -499,6 +508,7 @@ PYBIND11_MODULE(passes, m) {
       "performance. If squashing is disabled, each non-global PhasedX gate "
       "will be replaced with two global NPhasedX, but any other gates will "
       "be left untouched."
+      "\n\nDEPRECATED: This pass will be removed after pytket version 1.39."
```

Comment:
Do we want to put a specific version, or maybe better a more specific date?

Reply:
Intentionally put "after" so as not to be too committal, if we don't want to actually remove it.

Reply:
Maybe a date would be better though.